src/rate_limiter.cpp

```diff
@@ -65,6 +77,7 @@ bool memcached_rate_limiter::check(const std::string &key) {
   memcached_return error;
 
   mc_key = "cgimap:" + key;
+  int bytes_per_sec = moderator ? moderator_bytes_per_sec : standard_bytes_per_sec;
 
```

Comment:
Note on refactoring: we should probably move parsing of parameters, and setting of default values to the central location for options handling in options.[ch]pp. 

Then use something like `global_settings::get_bytes_per_sec(moderator)` and `global_settings::get_max_bytes(moderator)`  to fetch the relevant limits for the current user type.

Reply:
Done.